### PR TITLE
docs: add a row of activity_details to the table in README

### LINKED ISSUE #1256: [Docs] add a row of `activity_details` to the table in README
### Description

In #1187 and #1207, @frank-zsy added support for exporting activity_details.json for repos. The amazing data seems to be forgotten to put in [the README table](https://github.com/X-lab2017/open-digger#for-repos). 😆 I will make a codepen demo then update both the README and the official website docs. 

A data link: https://oss.x-lab.info/open_digger/github/X-lab2017/open-digger/activity_details.json

### PR DESCRIPTION
**Pull request by tyn1998**
resolve #1256 

preview: https://github.com/tyn1998/open-digger/blob/eb6ee0f247528175afad3998ff75a5935646259b/README.md

Demo：https://codepen.io/tyn1998/pen/KKGxVrm

image

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -202,6 +202,13 @@ Feel free to use the data to construct your own data application and you can ref
       <td><a href="https://github.com/X-lab2017/open-digger/blob/master/src/metrics/chaoss.ts#L494">Link</a></td>
       <td><a href="https://codepen.io/frank-zsy/pen/VwBqwaP?type=change_request_age">Demo</a></td>
     </tr>
+    <tr>
```
Comment (xgdyp) [inline docs/README.md:205]: hi, there may have some format problems, can you have a look?

Comment (xgdyp) [reply docs/README.md:205]: > hi, there may have some format problems, can you have a look? image

This should be in `docs/zh-cn/README.md`. I marked the wrong position

Comment (tyn1998) [reply docs/README.md:205]: That was a mistake :) fixed it.

**Comment by tyn1998** [comment]
A little improvement:

image

**Pull request merged**